chart/templates/deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
               fieldPath: metadata.name
         - name: ROX_SKIP_PEER_VALIDATION
           value: "true"
+        - name: ROX_RHCOS_NODE_SCANNING
+          value: "true"
         resources:
```

Comment:
I think this only would have an effect in `stackrox/scanner` CI. `stackrox/stackrox` deploys Scanner using its Helm chart. Therefore, environment value override should also be done there if you would like to see how this API functions when called from Central.